### DISCUSSION TITLE
Catch book automatically remembers last hunted reserve

### DISCUSSION THREAD

**Discussion by Errandir-of-Gondolin** [janstehno/cotw-companion#92] [Ideas]
It would save a few clicks if the catch book could remember what reserve you last made an entry from; most of the times, consecutive catches will be from the same reserve, having to scroll through the list to find the right one when making a new entry isn't the most efficient. If the catch book would automatically select the last used one, that would be a nice QoL improvement.

**Comment by janstehno** [comment]
It could remember every information instead of just the reserve. But yeah, the reserve seems to be the most convenient one.
